### ISSUE TITLE
'dicom2nifti' 不是内部或外部命令，也不是可运行的程序 或批处理文件。

### ISSUE THREAD

**Issue by ljljlj02** [icometrix/dicom2nifti#79]
When I run the  command----"dicom2nifti [-h] [-G] [-r] [-o RESAMPLE_ORDER] [-p RESAMPLE_PADDING] [-M] [-C] [-R]   my input data directory    my output data directory"    under  path----"\dicom2nifti-master\dicom2nifti\", there is a error "'dicom2nifti' 不是内部或外部命令，也不是可运行的程序 或批处理文件。"  Do you know why?

**Comment by abrys** [comment]
You should be running "dicom2nifti input_directory output_directory" and add any of the options if needed. To see what options do what do "dicom2nifti -h"
"[-h] [-G] [-r] [-o RESAMPLE_ORDER] [-p RESAMPLE_PADDING] [-M] [-C] [-R]" is just a list of the possible parameters but in general are not needed

**Comment by ljljlj02** [comment]
yeah,i know that,but when i run "dicom2nifti -h" under path----"\dicom2nifti-master\dicom2nifti", the error "'dicom2nifti' 不是内部或外部命令，也不是可运行的程序 或批处理文件。"also occur ...

**Comment by abrys** [comment]
Used google translate, can you please confirm that it translates to 
"Not an internal or external command, nor a runnable program or batch file"

Could you provide some more information:
version of python
operating system

One more remark, if you are familiar with conda i would advise to install dicom2nifti using conda.
https://anaconda.org/conda-forge/dicom2nifti

**Comment by ljljlj02** [comment]
it is right,"Not an internal or external command, nor a runnable program or batch file"
and my python version is 3.6.6,operating system is win10. i also  install dicom2nifti using conda.
 But mistakes remain
![1](https://user-images.githubusercontent.com/50227846/84234023-0f2dfb00-ab26-11ea-8187-acfa2b01d5e3.jpg)

Any help would be appreciated.

**Comment by abrys** [comment]
Can you try the following command in your conda env.
python -c  'import dicom2nifti; print(dicom2nifti.__file__)'

**Comment by ljljlj02** [comment]
I was able to successfully import dicom2nifti in conda Env.

**Comment by abrys** [comment]
I will be trying to run the command-line version of dicom2nifti on windows somewhere in the next couple days. Maybe something is not correct there causing it to not be found in the path.

**Comment by ljljlj02** [comment]
thank you ! I hope to hear good news from you :)

**Comment by abrys** [comment]
I had some time to test on windows and it seems that the way we created the command line version of dicom2nifti will not work on windows (only on mac and linux).

I have created a workaround or alternatively use the python interface if you are familiar with coding in python.

For the workaround use the anaconda power shell and do the following
`
# installation and apply workaround
conda create --name dicom2nifti
conda activate dicom2nifti
conda install -c conda-forge dicom2nifti
cd C:\Users\username\Miniconda3\envs\dicom2nifti
mv .\Scripts\dicom2nifti .\Scripts\dicom2nifti_tool

# from now on
conda activate dicom2nifti
cd C:\Users\username\Miniconda3\envs\dicom2nifti
.\python.exe .\Scripts\dicom2nifti_tool --help # or other parameters`

**Comment by abrys** [comment]
For an actual fix it will have to wait for quite some time as I would need to research this further and currently we have some other priorities. Hope this already helps you to get started.

**Comment by ljljlj02** [comment]
Fine.thank you :)
